openedx/features/partners/views.py

```diff
@@ -38,3 +42,21 @@ def register_user(request, slug):
         return views.Give2AsiaRegistrationView.as_view()(request, partner=partner)
     except ImportError:
         raise Http404('Your partner is not properly registered')
+
+
+@csrf_exempt
+@require_http_methods(['POST'])
```

Comment:
This would require a separate import, `require_http_methods` is already being used and does essentially the same thing.

Reply:
@zeshanarifios would you please discuss what could the benefits of using `@api_view()` here coz you recently added an api with this decorator. Actually it's  inconsistency in our code, at some points we are using it and at others, we aren't also it would be great if we can come up with some best practice as well.